corehq/apps/users/templates/users/partial/manage_phone_numbers.html

```diff
@@ -0,0 +1,113 @@
+{% load i18n %}
```

Comment:
Just moved this into its own template from users/partial/basic_info_form.html (first commit) and then make some minor tweaks to it (second commit)